src/main/java/org/itxtech/synapseapi/utils/DataPacketEidReplacer.java

```diff
@@ -112,38 +118,19 @@ private static EntityMetadata replaceMetadata(EntityMetadata data, long from, lo
         boolean changed = false;
 
         for (Integer key : replaceMetadata) {
-            try {
-                @SuppressWarnings("rawtypes")
-                EntityData ed = data.get(key);
-
-                if (ed == null) {
-                    continue;
-                }
-
-                if (ed.getType() != Entity.DATA_TYPE_LONG) {
-                    MainLogger.getLogger().info("Wrong entity data type (" + key + ") expected 'Long' got '" + dataTypeToString(ed.getType()) + "'");
-                    continue;
+            if (data.getLong(key) == from) {
+                if (!changed) {
+                    data = cloneMetadata(data);
+                    changed = true;
                 }
 
-                long value = ((LongEntityData) ed).getData();
-
-                if (value == from) {
-                    if (!changed) {
-                        data = cloneMetadata(data);
-                        changed = true;
-                    }
-
-                    data.putLong(key, to);
-                }
-            } catch (Exception e) {
-                MainLogger.getLogger().error("Exception while replacing metadata '" + key + "'", e);
+                data.putLong(key, to);
             }
         }
 
         return data;
     }
 
-    @SuppressWarnings("rawtypes")
```

Comment:
Why?

Reply:
you can add back

Reply:
What's the point of change something that will be changed back in next commit?